ksqldb-engine/src/test/java/io/confluent/ksql/analyzer/PullQueryValidatorTest.java

```diff
@@ -15,33 +15,42 @@
 
 package io.confluent.ksql.analyzer;
 
+import static io.confluent.ksql.schema.ksql.SystemColumns.ROWPARTITION_NAME;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.assertThrows;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
 import static org.mockito.Mockito.when;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
 import io.confluent.ksql.analyzer.Analysis.Into;
+import io.confluent.ksql.execution.expression.tree.ColumnReferenceExp;
 import io.confluent.ksql.execution.expression.tree.Expression;
-import io.confluent.ksql.parser.OutputRefinement;
+import io.confluent.ksql.execution.util.ColumnExtractor;
+import io.confluent.ksql.name.ColumnName;
 import io.confluent.ksql.parser.tree.GroupBy;
 import io.confluent.ksql.parser.tree.PartitionBy;
+import io.confluent.ksql.parser.tree.SingleColumn;
 import io.confluent.ksql.parser.tree.WindowExpression;
 import io.confluent.ksql.serde.RefinementInfo;
+import io.confluent.ksql.util.KsqlConfig;
 import io.confluent.ksql.util.KsqlException;
 import java.util.Optional;
 import java.util.OptionalInt;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
+import org.mockito.MockedStatic;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
 public class PullQueryValidatorTest {
 
   private static final Expression AN_EXPRESSION = mock(Expression.class);
+  private static final ColumnName A_BAD_COLUMN_NAME = ROWPARTITION_NAME;
```

Comment:
Let's remove this variable and use `ROWPARTITION_NAME` inline directly. I think the name `A_BAD_COLUMN_NAME` is actually more confusing since it's not clear what "bad" means.